testserver/testserver.go

```diff
@@ -497,7 +500,7 @@ func (ts *testServerImpl) Start() error {
 	if len(ts.stderr) > 0 {
 		wr, err := newFileLogWriter(ts.stderr)
 		if err != nil {
-			return fmt.Errorf("unable to open file %s: %s", ts.stderr, err)
+			return fmt.Errorf("testserver: unable to open file %s: %s", ts.stderr, err)
```

Comment:
we should wrap the error here rather than use `fmt.Errorf`

see https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/1676640931/Error+handling+basics
(although for this library we cannot use cockroachdb/errors, so we can use the golang `errors` package